src/Algebra/Properties/Group.agda

```diff
@@ -96,13 +121,24 @@ identity-unique {x} id = identityˡ-unique x x (proj₂ id x)
 
 inverseˡ-unique : ∀ x y → x ∙ y ≈ ε → x ≈ y ⁻¹
 inverseˡ-unique x y eq = begin
-  x              ≈⟨ left-helper x y ⟩
+  x              ≈⟨ rightDividesʳ y x ⟨
   (x ∙ y) ∙ y ⁻¹ ≈⟨ ∙-congʳ eq ⟩
        ε  ∙ y ⁻¹ ≈⟨ identityˡ (y ⁻¹) ⟩
             y ⁻¹ ∎
 
 inverseʳ-unique : ∀ x y → x ∙ y ≈ ε → y ≈ x ⁻¹
 inverseʳ-unique x y eq = begin
-  y       ≈⟨ sym (⁻¹-involutive y) ⟩
-  y ⁻¹ ⁻¹ ≈⟨ ⁻¹-cong (sym (inverseˡ-unique x y eq)) ⟩
+  y       ≈⟨ ⁻¹-involutive y ⟨
+  y ⁻¹ ⁻¹ ≈⟨ ⁻¹-cong (inverseˡ-unique x y eq) ⟨
   x ⁻¹    ∎
+
+\\≗//ᵒ⇒comm : (∀ x y → x \\ y ≈ y // x) → Commutative _∙_
+\\≗//ᵒ⇒comm \\≗//ᵒ x y = begin
+  x ∙ y                ≈⟨ identityʳ _ ⟨
+  (x ∙ y) ∙ ε          ≈⟨ ∙-congˡ (inverseˡ x) ⟨
+  (x ∙ y) ∙ (x ⁻¹ ∙ x) ≈⟨ assoc (x ∙ y) (x ⁻¹) x ⟨
+  (x ∙ y) ∙ x ⁻¹ ∙ x   ≈⟨ ∙-congʳ (assoc x y (x ⁻¹)) ⟩
+  x ∙ (y // x) ∙ x     ≈⟨ ∙-congʳ (∙-congˡ (\\≗//ᵒ x y)) ⟨
```

Comment:
Isn't this the proof `rightDivides` run backwards?

Reply:
Ha! Entirely possible, but for having banged my head at each of these fur a while trying to eliminate redundancy, until I could no longer see the wood for the trees... will try to look with fresh eyes later or tomorrow.

Reply:
Beginning to be convinced!